City-of-Helsinki/youth-membership-ui
# YM-419 | Release/1.2.0

## Description

This PR contains the latest changes from develop branch.

## Context

[YM-419](https://helsinkisolutionoffice.atlassian.net/browse/YM-419)

## How Has This Been Tested?

- Automatic E2E tests have been created and run locally.
- Manual tests have been run during the development of features.

## Manual Testing Instructions for Reviewers

See the individual tickets of the release in Jira: https://helsinkisolutionoffice.atlassian.net/browse/YM/fixforversion/10074

## Screenshots

N/A
